src/content/en/fundamentals/performance/navigation-and-resource-timing/index.md

```diff
@@ -0,0 +1,689 @@
+performance. Fortunately, both APIs expose properties for quantifying payloads:
+
+- `**transferSize**` is the _total size_ of the resource including HTTP headers.
+- `**encodedBodySize**` is the _compressed size_ of the resource _excluding_ HTTP
+headers.
+- `**decodedBodySize**` is the _decompressed size_ of the resource (also excluding
+HTTP headers).
+
+Using simple arithmetic, it's easy to figure out stuff like how much of a
+response consists of HTTP headers or even compression ratios:
+
+```javascript
+// HTTP header size
+var pageNav = performance.getEntriesByType("navigation")[0];
+var headerSize = pageNav.transferSize - pageNav.encodedBodySize;
+
+// Compression ratio
+var compressionRatio = pageNav.decodedBodySize / pageNav.encodedBodySize;
+```
+
+You'll likely already be aware of resource size, and you don't really need an
+API to tell you this stuff, as the network panel in any browser's dev tools
+conveys this information. But the information is there if you need it!
+
+## Acquiring timings in application code
+
+Now that you know some of the metrics provided by these APIs, let's cover how
+you can collect that data in your application code.
+
+### Other methods for manually grabbing timings
+
+Earlier in this guide, you dabbled with `getEntriesByType`, which is great for
+grabbing performance entries of a specific type. There are two other related
+methods worth mentioning, however.
+
+#### `getEntriesByName`
+
+[`getEntriesByName`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntriesByName)
+gets a performance entry by its name. For Navigation and Resource Timing, this
+is the URL for the document or resource:
+
+```javascript
+// Get timing data for an important hero image
+var heroImageTime = performance.getEntriesByName("https://somesite.com/images/hero-image.jpg");
+```
+
+This is _super_ useful if you need to get a performance entry for a single, yet
+vital, resource. It's also a better performing alternative to filtering out an
+array in a loop or with a reduction function.
+
+#### getEntries
+
+Unlike `getEntriesByName` and `getEntriesByType`,
+[`getEntries`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/getEntries)
+gets everything in the performance entry buffer by default:
+
+```javascript
+// Get timing data for an important hero image
+var allTheTimings = performance.getEntries();
+```
+
+But wait, there's more! `getEntries` can also serve as more verbose alternatives
+to `getEntriesByName` and `getEntriesByType`:
+
+```javascript
+// Get timing data for an important hero image
+var allTheTimings = performance.getEntries({
+  // Get entries by name
+  "name": "https://somesite.com/images/hero-image.jpg",
+  // Get entries by type
+  "entryType": "resource",
+  // Get entries by their initiatorType value:
+  "initiatorType": "img"
+});
+```
+
+If `initiatorType` looks new to you, that's because I didn't cover it. You can
+find out more about it in [this MDN
```

Comment:
Suggestion: "Find out more about it in MDN's [Performance Resource Timing `initiatorType`](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/initiatorType) docs."

Reply:
Same for other links throughout the doc